docs/architecture/clients/index.md

```diff
@@ -10,18 +10,65 @@ and a single [Git repository](https://github.com/bitwarden/clients).
 
 The mono-repository root directory contains three main folders.
 
-- `apps` - Our different application specific code, consists of `web`, `browser`, `desktop` and
-  `cli`.
-- `bitwarden_license` - Bitwarden Licensed version of the web vault.
-- `libs` - Shared code between the different applications.
-
-`libs` contains the following projects.
-
-- `Common` - Common code shared between all the clients including CLI.
-- `Angular` - Angular specific code used by all the visual clients.
-- `Components` - Angular Components Library.
-- `Node` - Used to be shared code for CLI and Directory Connector CLI, but since directory connector
-  no longer uses the same version of libs this module is deprecated.
+- `apps`: Our different application specific code, consists of `web`, `browser`, `desktop` and
+  `cli`
+- `bitwarden_license`: Bitwarden Licensed version of the web vault
```

Comment:
While currently only the web vault it might cover other areas later.

```suggestion
- `bitwarden_license/web-vault`: Bitwarden Licensed version of the web vault
```